src/applications/gi/utils/helpers.js

```diff
@@ -88,3 +90,34 @@ export const formatDollarAmount = value => {
   const output = value != null ? removeNonNumberCharacters(value) : 0;
   return formatCurrency(output);
 };
+
+export const handleInputFocus = fieldId => {
+  const field = document.getElementById(fieldId);
+  if (field && window.innerWidth <= SMALL_SCREEN_WIDTH) {
+    // prod flag for bah-8821 EYB changes
+    if (environment.isProduction()) {
+      field.scrollIntoView();
+    } else {
+      field.scrollIntoView({ behavior: 'smooth', block: 'nearest' });
+      const eybSheetField = document.getElementById('eyb-sheet');
```

Comment:
this is specific to `src/applications/gi/containers/EstimateYourBenefits.jsx` and not all of CT and should be a function in that container and passed into it's child components if needed by them